common_sleep.Rmd

```diff
@@ -200,224 +230,271 @@ lattice::xyplot(Reaction ~ Days|Sx,
+
+cowplot::plot_grid(hist_intercept, hist_slope, hist_rsquared, 
+                   nrow = 1,
+                   labels = "AUTO")
 ```
 
-The individual intercepts shown in the first histogram correspond to the initial reaction time and are non normally distributed. Given the small data set, this is not surprising as it shows a variety of their initial reaction time. However, if this data came from a large dataset, it would be surprising that even the initial data points are not normally distributed and could suggest a wrong data sample compared to the population. 
 
-Looking at the histogram of individual slopes, we see a normal distribution. As seen on the previous graph showcasing the individual linear regressions, very little slopes are negative. This shows again that reaction time increases by days of sleep deprivation. 
+The individual intercepts shown in the first histogram (A) correspond to the initial reaction time at day zero and are non normally distributed. Given the small data set, this is not surprising as it shows a variety of the initial reaction time. However, if this data came from a large dataset, it would be surprising that the initial data points are not normally distributed and could suggest a wrong data sample compared to the population. 
+
+Looking at the histogram of individual slopes (B), we see a normal distribution. As seen on the previous graph showcasing the individual linear regressions, only one slope is negative. This shows again that reaction time increases by days of sleep deprivation. 
 
-Finally, looking at the histogram of R squared, we see that the majority of subjects have a R squared of above 0.6. This shows that the linear model is appropriate for this data set. However, sometimes the individual linear model does not fit the specific data of some subjects, specifically 7 of the 18 subjects.
+Finally, looking at the histogram of R squared (C), we see that the majority of subjects have a R squared of above 0.6. This shows that the linear model is appropriate for this data set. However, the individual linear model does not fit the specific data of some subjects, respectively 7 of the 18 subjects.
 
 ## Fitting the model - with REML
-```{r}
-sleep.reml <- lmer(formula = Reaction ~ 1 + Days + (1 + Days|Subject),
-                   data = sleep)
-summary(sleep.reml)
-```
-## Values
-$$\begin{aligned}
-\gamma_{0}  &= 251.405 \\
-\gamma_{1}  &= 10.467  \\
-\\
-\sigma_{\epsilon}^{2} &= 654.94 \\
-\sigma_{0}^{2} &= 611.90 \\
-\sigma_{1}^{2} &= 35.08 \\
-corr(b_{0i}, b_{1i}) &= 0.07
-\end{aligned}$$
 
-##Mathematical description
+## Mathematical description
 
-Level 1 model explains the evolution of Reaction time for each subject
+Level 1 model explains the evolution of Reaction time for each subject:
 $$\begin{aligned}
 Y_{ij}&= \pi_{0i} + \pi_{1i}* \text{Days}_{ij} && \text{how do individuals evolve} \\
           &+ \epsilon_{ij} &&\text{how the individuals deviate from their own evolution}
```

Comment:
I don't understand what this sentence means... can we just refer to it as random error?